### DISCUSSION TITLE
Get geoid height at specific lat and lon values

### DISCUSSION THREAD

**Discussion by danielissac** [pyproj4/pyproj#1235] [Q&A] [answered]
Hey folks,
I want to know how to get geoid height for the specific latitude and longitude values using pyproj.
Example:
Inputs:
latitude - 35.6878962
longitude - 139.7723618
Output:
Geoid height - 36.6912

**Comment by piyushrpt** [comment, marked as answer]
See this - https://github.com/pyproj4/pyproj/issues/630

**Reply by danielissac** [reply]
Thanks for the link. I'm totally new to GIS and pyproj. However I tried the below snippet.
```
import pyproj
pyproj.network.set_network_enabled(active=True)
tr = pyproj.transformer.Transformer.from_crs(6667,6697)
print(tr.transform(35.6878962,139.7723618,0.0))
```
I got the expected output of 36.6912. 
But when I try with different latitude and longitude values (26.63333,127.96667,0.0). I got the output of 0 but the expected output is 32.8603. I'm just confused and am I missing something here?

Actually I am following this [link](https://qiita.com/tohka383/items/e8392db87359e384a91e) to find out the geoid height for Japan location and I want to achieve something like [this](https://vldb.gsi.go.jp/sokuchi/surveycalc/geoid/calcgh/calcframe.html) with pyproj.

**Reply by piyushrpt** [reply]
Check the bounds on https://epsg.io/6697 - I don't think it is defined for the lat/lon that you are looking at.

**Reply by danielissac** [reply]
Thanks. I got it. one more thing how do I get rid of the below line.
`pyproj.network.set_network_enabled(active=True)`
what this line exactly doing? because without this I'm not getting the values.

**Reply by piyushrpt** [reply]
If you download the grid locally - you don't need network access. Look at the discussion at the end of that thread I pointed you to. 
https://pyproj4.github.io/pyproj/stable/transformation_grids.html

**Reply by danielissac** [reply]
Got it. Thanks for the help 🙇‍♂️.
